core/control-plane/control-plane-aggregate-services/build.gradle.kts

```diff
@@ -17,14 +17,15 @@ plugins {
 }
 
 dependencies {
-    implementation(project(":spi:common:validator-spi"))
-    implementation(project(":spi:control-plane:control-plane-spi"))
+    api(project(":spi:common:boot-spi"))
```

Comment:
Why are these exposed as APIs? They are not needed to extend this module.

Reply:
must have been some rebase/pr split issue, will remove it

Reply:
I think a module should only expose another module as an `api` if it is transitively exposed to interact with it, e.g. an SPI interface references a type from the other module.